config/environments/development.rb

```diff
@@ -90,4 +90,7 @@
   # Use a real queuing backend for Active Job (and separate queues per environment).
   config.active_job.queue_adapter = :solid_queue
   # config.active_job.queue_name_prefix = "ecf2_development"
+
+  # Custom config
+  config.bypass_filter_parameter_logging = ENV.fetch("ENABLE_BYPASS_FILTER_PARAMETER_LOGGING", false)
```

Comment:
I wonder if it's worth hardcoding this to `false` in `config/application.rb` but allowing it to be overridden here?

Then we wouldn't need the slightly-complex `if` statement in `config/initializers/filter_parameter_logging.rb`, so it could be:

```ruby
Rails.application.config.filter_parameters += %i[_key]

if Rails.application.config.bypass_filter_parameter_logging
  Rails.application.config.filter_parameters += other_stuff
end
```

Reply:
Can't think of anything to add other than maybe sourcing the list from a YAML file.